scripts/check_evil_merge.py

```
@@ -0,0 +1,262 @@
+        return []
+
+    merge_sha = _run_git(["rev-parse", head_ref], cwd=repo_root).strip()
+    parent1 = parents[0]
+    parent2 = parents[1]
+
+    head_paths = _get_test_files(repo_root, head_ref)
+    head_blobs = _get_blob_hashes(repo_root, head_ref, head_paths)
+    p1_blobs = _get_blob_hashes(repo_root, parent1, head_paths)
+    p2_blobs = _get_blob_hashes(repo_root, parent2, head_paths)
+
+    merge_tree_result = subprocess.run(
```

Ruff (0.16.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)